### DISCUSSION TITLE
color class does seem to work?

### DISCUSSION THREAD

**Discussion by murugappanrm** [tailwindlabs/tailwindcss#8528] [Help]
Version: Latest version of PlayCDN.

I entered the following code:

```
<p class="text-red">This is red text</p>
```

The text color remained as default and did not change to red. But when i entered the same code with opacity, the color works. It looks like, other than the colors white and black, other colors do have a default opacity.

Is this a bug or design flaw? Please advise.

**Comment by adamwathan** [comment]
Hey! There is no `text-red` class in Tailwind, only classes like `text-red-500`, `text-red-200`, etc.

**Comment by murugappanrm** [comment]
@adamwathan 
Sorry my post has a typo error. It should be :

I entered the following code:

<p class="text-red">This is red text</p>

The text color remained as default and did not change to red. But when i entered the same code with opacity, the color works. It looks like, other than the colors white and black, other colors **do not** have a default opacity.
